Snakemake/workflow/rules/prepare_files_for_tsinfer.smk

```diff
@@ -48,7 +48,12 @@ rule extract_vcf_pos:
     input: rules.get_major.output
         #rules.decompress.output
     output:
+<<<<<<< HEAD
```

Comment:
There are some cofnlicts here, that is why you have this "HEAD". This needs to be cleaned before merging